http://dev.mutt.org/trac/changeset/1cf34ea1f128
http://dev.mutt.org/trac/ticket/3356

STACK* is not defined with newer SSL, use STACK_OF instead.
Closes #3356.

Index: mutt_ssl.c
===================================================================
--- mutt_ssl.c (revision 6016:dc09812e63a3)
+++ mutt_ssl.c (revision 6031:1cf34ea1f128)
@@ -653,5 +653,5 @@
   int bufsize;
   /* needed to get the DNS subjectAltNames: */
-  STACK *subj_alt_names;
+  STACK_OF(GENERAL_NAME) *subj_alt_names;
   int subj_alt_names_count;
   GENERAL_NAME *subj_alt_name;
